### PR TITLE
[nearlib] a few test infrastructure fixes+ pass base url to connection

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: nearlib/signing/browser_local_storage_keystore.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        window.localStorage.setItem(
+            BrowserLocalStorageKeystore.storageKeyForPublicKey(accountId), key["public_key"]);
+        window.localStorage.setItem(
+            BrowserLocalStorageKeystore.storageKeyForSecretKey(accountId), key["secret_key"]);
+    };
+
+    async getKey(accountId) {
+        console.log(window.localStorage.getItem(
+            BrowserLocalStorageKeystore.storageKeyForPublicKey(accountId)));
+        console.log(window.localStorage.getItem(
```
Comment (vgrichina) [inline nearlib/signing/browser_local_storage_keystore.js]: let's not log private keys

File: nearlib/signing/browser_local_storage_keystore.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * browser sessions. Local storage likes to work with strings so we store public and private key separately.
+ */
+
+const LOCAL_STORAGE_SECRET_KEY_SUFFIX = "_public";
+const LOCAL_STORAGE_PUBLIC_KEY_SUFFIX = "_secret";
+
+class BrowserLocalStorageKeystore {
+    constructor() {}
+
+    static storageKeyForPublicKey(accountId) {
```
Comment (vgrichina) [inline nearlib/signing/browser_local_storage_keystore.js]: why not store whole json under one key?

File: nearlib/nearclient.js [hunk 3]
```diff
@@ -47,6 +48,21 @@ class NearClient {
...
         return await this.nearConnection.request(methodName, params);
     };
+
+    async generateNewKeyFromRandomSeed () {
+        var randomSeed = crypto.randomBytes(32);
+        var newKeypair = ed25519.MakeKeypair(randomSeed);
+        const response = {};
+        response['public_key'] = this.encodeBufferInBs58(newKeypair.publicKey);
+        response['secret_key'] = this.encodeBufferInBs58(newKeypair.privateKey);
+        return response;
```
Comment (vgrichina) [inline nearlib/nearclient.js]:
```js
return {
    public_key: this.encodeBufferInBs58(newKeypair.publicKey),
    secret_key: this.encodeBufferInBs58(newKeypair.privateKey)
} 
```

Comment (vgrichina) [reply nearlib/nearclient.js]: same applies to other cases. Very preferable to construct object literal vs assign properties one by one.

**Comment by janedegtiareva** [comment]
I am going to revisit comments after hackathon

**Pull request merged**